app/tests/cases_tests/test_tiff.py

```diff
@@ -65,6 +70,48 @@ def test_tiff_validation(resource, expected_error_message):
     assert error_message == expected_error_message
 
 
+@pytest.mark.parametrize(
+    "source_dir, filename, expected_error_message",
+    [
+        (RESOURCE_PATH, "valid_tiff.tif", ""),
+        (RESOURCE_PATH, "image5x6x7.mhd", "Image isn't a TIFF file"),
+        (
+            RESOURCE_PATH,
+            "invalid_meta_data_tiff.tif",
+            "Image contains unauthorized information",
+        ),
+        (
+            RESOURCE_PATH,
+            "invalid_resolutions_tiff.tif",
+            "Image only has a single resolution level",
+        ),
+        (
+            RESOURCE_PATH,
+            "invalid_tiles_tiff.tif",
+            "Image has incomplete tile information",
+        ),
+    ],
+)
+def test_dzi_creation(
+    source_dir, filename, expected_error_message, tmpdir_factory
+):
+    error_message = ""
+
+    try:
```

Comment:
A more pythonic way to do this would be
```
with pytest.raises(ValidationError) as err:
    ...

assert err.message == expected_message
```

Reply:
Also, remove the lines that are not expected to raise the error out of the `try` or `with` block

Reply:
In that case I would have to split the test, because the first (valid) file that is tested doesn't raise an error. I could, make a separate test for valid files, and one for invalid files.